pdr_backend/lake/etl.py

```diff
@@ -1,28 +1,27 @@
 import logging
 import time
 from typing import Dict, List, Optional, Tuple
+
 from enforce_typing import enforce_types
 
-from pdr_backend.ppss.ppss import PPSS
-from pdr_backend.lake.table import TableType, get_table_name, NamedTable, TempTable
 from pdr_backend.lake.gql_data_factory import GQLDataFactory
+from pdr_backend.lake.persistent_data_store import PersistentDataStore
+from pdr_backend.lake.table import NamedTable, TableType, TempTable, get_table_name
 from pdr_backend.lake.table_bronze_pdr_predictions import (
-    bronze_pdr_predictions_table_name,
-    bronze_pdr_predictions_schema,
+    BronzePrediction,
     get_bronze_pdr_predictions_data_with_SQL,
 )
 from pdr_backend.lake.table_bronze_pdr_slots import (
-    bronze_pdr_slots_table_name,
-    bronze_pdr_slots_schema,
+    BronzeSlot,
     get_bronze_pdr_slots_data_with_SQL,
 )
 from pdr_backend.lake.table_registry import TableRegistry
-from pdr_backend.lake.persistent_data_store import PersistentDataStore
-from pdr_backend.lake.table_pdr_payouts import payouts_table_name
-from pdr_backend.lake.table_pdr_predictions import predictions_table_name
-from pdr_backend.lake.table_pdr_subscriptions import subscriptions_table_name
-from pdr_backend.lake.table_pdr_slots import slots_table_name
-from pdr_backend.lake.table_pdr_truevals import truevals_table_name
+from pdr_backend.ppss.ppss import PPSS
+from pdr_backend.subgraph.payout import Payout
+from pdr_backend.subgraph.prediction import Prediction
+from pdr_backend.subgraph.slot import Slot
+from pdr_backend.subgraph.subscription import Subscription
+from pdr_backend.subgraph.trueval import Trueval
```

Comment:
I believe the tables should exist in `lake/` not `subgraph/`

Reply:
for etl models I agree. For raw models, the fetch functions return these object types, as defined in subgraph. Essentially, ETL tables copy subgraph objects at a higher level. They exist in subgraph not because I put them there, but because they are at a more basic level. They were already there, I just added the functions to retrieve lake schema and lake table name on them.

Reply:
TLDR: raw tables are not ETL tables, they are reflections of subgraph

Reply:
update: I've checked. Actually they are pretty "movable" primitives and there are even name clashes in subgraph. I've moved them and I'll re-request a review once the tests pass.